module/purpose/Soccer/data/config/webots/Soccer.yaml

```diff
@@ -0,0 +1,9 @@
+# Controls the minimum log level that NUClear log will display
+log_level: INFO
+
+# For the robot playing without listening to GameController
+force_playing: false
+
+# The position of the robot
+# GOALIE, STRIKER, DEFENDER
+position: STRIKER
```

Comment:
Is this relevant to this PR?

Reply:
Not entirely, but it got removed when it shouldn't have in another PR prior and is needed to test the filter in webots